xbmc/cores/RetroPlayer/cheevos/Cheevos.cpp

```diff
@@ -160,3 +212,20 @@ RConsoleID CCheevos::ConsoleID()
 
   return it->second;
 }
+
+void CCheevos::Callback_URL_ID(const char* achievementUrl, unsigned int cheevoId)
+{
+  XFILE::CCurlFile curl;
+  std::string res;
+  curl.Get(achievementUrl, res);
+  std::string description = m_activatedCheevoMap[cheevoId][1];
+  std::string header = std::string("Congratulations, ") + std::string("Achievement Unlocked");
```

Comment:
any reason why `std::string` is explicit here? and why adding two static strings together?